### PR TITLE
MAGStock 7: Updating camping_types

### PR DESCRIPTION
**Pull request by Hachi05**
Added an option for cabins among other changes/enhancements.

### CODE REVIEW

File: event-magstock.yaml [hunk 1]
```diff
@@ -124,10 +124,13 @@ uber::plugin_magstock::site_types:
   normal:     "Normal - has electric and water hookups"
 
 uber::plugin_magstock::camping_types:
-  small_tent:  "Small Tent (6 or fewer)"
-  large_tent:  "Large Tent (more than 6)"
-  car:         "Sleeping in car"
-  rv:          "RV or other larger vehicle (Additional fees may apply; please contact us)"
+  small_tent:  "Small Tent (small footprint)"
+  medium_sent: "Medium Tent (around 4 people)"
```
Comment (kitsuta) [inline event-magstock.yaml]: Typo: `medium_sent`.

File: event-magstock.yaml [hunk 2]
```diff
@@ -124,10 +124,13 @@ uber::plugin_magstock::site_types:
...
+  large_tent:  "Large Tent (Part City)"
+  car:         "Car - I plan to sleep in my car at my site for an additonal fee."
+  rv:          "RV - I plan to sleep in an RV or similar for an additonal fee."
+  special_site: "Special - I plan to do anything else at a campsite that will take up a lot of space."
+  cabin:      "Cabin - I plan to be boring and sleep in a cabin."
```
Comment (kitsuta) [inline event-magstock.yaml]: Although I won't block the PR based on this, I would reconsider the wording here. Insulting people while they're filling out a form is usually not a good idea. If I was a band and had to be in a cabin so my equipment wouldn't get wrecked, I'd be kind of annoyed that you thought it was funny to take a dig at me for it.

File: event-magstock.yaml [hunk 3]
```diff
@@ -124,10 +124,13 @@ uber::plugin_magstock::site_types:
   normal:     "Normal - has electric and water hookups"
 
 uber::plugin_magstock::camping_types:
-  small_tent:  "Small Tent (6 or fewer)"
-  large_tent:  "Large Tent (more than 6)"
-  car:         "Sleeping in car"
-  rv:          "RV or other larger vehicle (Additional fees may apply; please contact us)"
+  small_tent:  "Small Tent (small footprint)"
+  medium_sent: "Medium Tent (around 4 people)"
+  large_tent:  "Large Tent (Part City)"
```
Comment (kitsuta) [inline event-magstock.yaml]: What does "Part City" mean here?

**Review by kitsuta — COMMENTED** [review]
This is fine overall -- I found a typo and had some wording questions/suggestions, but that's just, like, my opinion, man.

The tricky part about changing these options this late in the game is that most people won't really go back and change their selection, so the data will be inconsistent. I'm figuring that you know that and mostly want these new options for admins.

**Comment by Hachi05** [comment]
alright, this should be good to go!

**Pull request merged**